tests/integration/security/https_jwt/main_test.go

```diff
@@ -0,0 +1,80 @@
+//go:build integ
+// +build integ
+
+//  Copyright Istio Authors
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package security
+
+import (
+	"testing"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/resource"
+	"istio.io/istio/tests/integration/security/util"
+)
+
+var (
+	ist  istio.Instance
+	apps = &util.EchoDeployments{}
+)
+
+func TestMain(m *testing.M) {
+	framework.
+		NewSuite(m).
+		Setup(istio.Setup(&ist, setupConfig)).
+		Setup(func(ctx resource.Context) error {
+			return util.SetupApps(ctx, ist, apps, true)
+		}).
+		Run()
+}
+
+func setupConfig(ctx resource.Context, cfg *istio.Config) {
+	if cfg == nil {
+		return
+	}
+
+	// command to generate certificate
+	// use the generated ca.crt by following https://github.com/istio/istio/blob/master/samples/jwt-server/testdata/README.MD
+	cfg.ControlPlaneValues = `
+values:
+  pilot: 
+    jwksResolverExtraRootCA: |
+      -----BEGIN CERTIFICATE-----
```

Comment:
nit: would be slightly nicer to read from file, so its not copied

Reply:
I guess we can't read it from file afaik, we are creating yaml template inside golang.

Reply:
+1 it's better to read from the file instead of hard coding the cert.

Reply:
I tried doing that way, but formatting got messed up which resulted in errors.

Reply:
this should be doable using the go template and the indent function, check the example here: https://github.com/istio/istio/blob/master/tests/integration/pilot/locality_test.go#L75